### DISCUSSION TITLE
Unclear which version of TonyHax I should use (NSTC SCPH-101)

### DISCUSSION THREAD

**Discussion by smvb64** [brad-lin/FreePSXBoot#74] [Q&A] [answered]
Hello all,

I am ready to mod my  NSTC SCPH-101 Psone using  FreePSXBoot and TonyHax but I am unsure on which version I should use.
I see there is a 4.4 and a 4.5 bios version.   Does it matter which version I use?

Thanks
Dan

**Comment by brad-lin** [comment, marked as answer]
Hello,

The version of your BIOS does matter. You can either:
* Try the two images one by one, assuming you can easily reformat the memory card if the first try fails
* Use Tonyhax with an original game to boot a Unirom boot disc, which will detect your BIOS and flash the memory card with the correct version. You can download the Unirom boot disc here: https://github.com/JonathanDotCel/unirom8_bootdisc_and_firmware_for_ps1/releases/tag/8.0.J
* You can also boot the Unirom boot disc using the disc swap trick, but it may be difficult to do on a PSOne

**Reply by smvb64** [reply]
Hello,

Thank you - interestingly enough - it booted with the flashed 4.5 memory card into Tonyhax. (slot 2_
Tonyhax is reporting that I have 4.4 bios - I'm going to flash that version. 
Even though it booted into TonyHax it just can't read a burnt disk -  Disc Error type D Code 12
I am hoping the correct bios will help
